data/product.d/rhev.conf

```diff
@@ -12,17 +12,20 @@ default_partitioning =
     /              (min 6 GiB)
     /home          (size 1 GiB)
     /tmp           (size 1 GiB)
-    /var           (size 15 GiB)
+    /var           (size 5 GiB)
+    /var/crash     (size 10 GiB)
```

Comment:
I think the unit tests are failing because `/var/crash` is not defined in `VIRTUALIZATION_PARTITIONING`.

Reply:
ok, preparing a fix

Reply:
done